src/parser/parser.h

```diff
@@ -83,7 +83,7 @@ extern "C"
 
   extern PT_NODE *parser_create_node (const PARSER_CONTEXT * parser);
   extern PT_NODE *parser_new_node (PARSER_CONTEXT * parser, PT_NODE_TYPE node);
-  extern PT_NODE *parser_init_node (PT_NODE * node);
+  extern PT_NODE *parser_init_node (PT_NODE * node, PT_NODE_TYPE node_type, bool bFirst, bool bKeepLnCn);
```

Comment:
naming convention: underscore(`_`) should be used for variable
e.g) `bool b_first` instead of `bool bFirst`

Reply:
Yes, thank you.
I will separate the function and rewrite it.